azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -89,19 +89,15 @@ def __init__(self, endpoint):
         self.status_last_timestamp = None
         self._version = FlexibleVersion("0.0.0.0")  # Version 0 means "unknown"
         self._supports_vm_settings = None   # Tri-state variable: None == Not Initialized, True == Supports, False == Does Not Support
-        self._supports_vm_settings_next_check = datetime.datetime.now()
+        self._supports_vm_settings_next_check = datetime.datetime.now(UTC)
         self._vm_settings_error_reporter = _VmSettingsErrorReporter()
         self._cached_vm_settings = None  # Cached value of the most recent vmSettings
 
         # restore the state of Fast Track
-        if not os.path.exists(self._get_fast_track_state_file()):
-            self._supports_vm_settings = False
-            self._supports_vm_settings_next_check = datetime.datetime.now()
-            self._fast_track_timestamp = timeutil.create_timestamp(datetime.datetime.min)
-        else:
-            self._supports_vm_settings = True
-            self._supports_vm_settings_next_check = datetime.datetime.now()
-            self._fast_track_timestamp = HostPluginProtocol.get_fast_track_timestamp()
+        self._supports_vm_settings = os.path.exists(self._get_fast_track_state_file())
+        self._fast_track_timestamp = HostPluginProtocol.get_fast_track_timestamp()
```

Comment:
get_fast_track_timestamp already does the "if not os.path.exists(self._get_fast_track_state_file()):", so I refactored it out of the if statement